org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/dom/DOMElement.java

```diff
@@ -377,6 +377,13 @@ public boolean isStartTagClosed() {
 	public boolean isEndTagClosed() {
 		return getEndTagCloseOffset() != NULL_VALUE;
 	}
+
+	/**
+	 * Returns true if element contains only DOMText
+	 */
+	public boolean containsTextOnly() {

```

Comment:
Please remove this method in static method in DOMUtils for instance.

Reply:
You mean, I should add it into DOMUtils as a static method?

Reply:
and remove it from DOMelement.java?

Reply:
yes it's exactly that. We should avoid adding new method (if it used once time) in DOMElement which is an API. If we see that it's very used, we could move to DOMElement again.

Reply:
Oops, my bad. We spoke about this in yesterday's meeting